platform/features/imagery-timeline/bundle.js

```diff
@@ -0,0 +1,38 @@
+define([
+	'openmct',
+	'./src/controllers/ImageryTimelineController',
+	
+], function (
+	openmct,
+	ImageryTimelineController
+) {
+	openmct.legacyRegistry.register("platform/features/imagery-timeline", {
+		"name" : "Imagery Timeline",
+		"description": "Provides a timeline view of imagery telemetry.",
+		"extensions": {
+			"views": [
+				{
+					"name": "Imagery Timeline",
+					"key": "imagery.timeline",
+					"cssClass": "icon-image",
+					"templateUrl": "templates/timeline.html",
+					"needs": [ "telemetry" ],
+					"delegation": true
+				}
+			],
+			"stylesheets": [
+				{
+					"stylesheetUrl": "css/timeline.css"
+				}
+			],
+			"controllers": [
+				{
+					"key": "ImageryTimelineController",
+                    "implementation": ImageryTimelineController,
```

Comment:
you'll wanna run `gulp checkstyle` and `gulp jscs` relatively often, they'll catch style errors for you.